catalog/NEON/NEON_CHM.jsonnet

```diff
@@ -0,0 +1,169 @@
+	Citation: Please use the appropriate citation(s) in your publications; see Terms of Use for more info.
+	Documentation: [NEON DP3.30015.001 Quick Start Guide](https://data.neonscience.org/api/v0/documents/quick-start-guides/NEON.QSG.DP3.30015.001v1?inline=true&fallback=html)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'airborne',
+    'ecology',
+    'lidar',
+    'neon',
+  ],
+  providers: [
+    ee.producer_provider('NEON', 'https://www.neonscience.org/'),
+    ee.host_provider(self_ee_catalog_url),
+  ],
+  extent: ee.extent(-170, 16, -66, 73,'2013-01-01T00:00:00Z',null),
+  summaries: {
+    'gee:schema': [
+      {
+        name: 'AOP_VISIT_NUMBER',
+        description: 'Unique visit number to the NEON site',
+        type: ee_const.var_type.string,
+      },     
+      {
+        name: 'FLIGHT_YEAR',
+        description: 'Year the data were collected',
+        type: ee_const.var_type.string,
+      },  
+      {
+        name: 'NEON_DOMAIN',
+        description: 'NEON eco-climatic domain code, "D01" to "D20". See https://www.neonscience.org/field-sites/about-field-sites',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_SITE',
+        description: 'NEON four-digit site code. See https://www.neonscience.org/field-sites',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_ID',
+        description: 'NEON data product identification code, "DP3.30015.001"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'NEON_DATA_PROD_URL',
+        description: 'NEON data product url: "https://data.neonscience.org/data-products/DP3.30015.001"',
+        type: ee_const.var_type.string,
+      },
+	  {
+        name: 'PRODUCT_TYPE',
+        description: 'Acronym of the NEON data product type; CHM = Canopy Height Model',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'SENSOR_NAME',
+        description: 'Make and model of the lidar sensor: "Optech Galaxy Prime","Optech Gemini","Riegl Q780"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'SENSOR_SERIAL',
+        description: 'Serial number of the lidar sensor: "12SEN311","11SEN287","5060445","220855"',
+        type: ee_const.var_type.string,
+      },
+      {
+        name: 'RELEASE_TAG',
+        description: 'NEON Release Tag. See https://www.neonscience.org/data-samples/data-management/data-revisions-releases',
+        type: ee_const.var_type.string,
+      },
+    ],
+	gsd: [
+      1.0,
+    ],
+    platform: [
+      'NEON',
+    ],
+    instruments: [
+      'Optech Galaxy Prime','Optech Gemini','Riegl Q780'
+    ],
+    'eo:bands': [
+	 {
+	  name:'CHM',
+	  description:'Canopy Height Model',
+	  'gee:units':'meter',
+	 },
+    ],
+    'gee:visualizations': [
+      {
+        display_name: 'CHM',
+        lookat: {
```

Comment:
Tighten this up and use lon, lat to match [Map.setCenter](https://developers.google.com/earth-engine/apidocs/map-setcenter)

```
        lookat: {lon: -119.25, lat: 37.06, zoom: 12},
```